### DISCUSSION TITLE
Please bring back repo level projects!

### DISCUSSION THREAD

**Discussion by ekolis** [community/community#30241] [Other Feature Feedback, Questions, & Ideas]
And make repo projects the default when I go to link an issue to a project. I don't want to spend all day scrolling through every project for every repo I own to do so!

**Comment by StevenVanRosendaal** [comment]
Yes please. I only found out just now because my students pointed it out that the old way of doing things in projects was no longer working. So i tried to see what the new projects have to offer, and i found this to be incredibly annoying within minutes of trying it out. Either bring back repo level projects, or give us an option to select a default repo voor created issues.
